test/OWNERS

```diff
@@ -1,7 +1,10 @@
 approvers:
-  - Ark-kun
-  - IronPan
-  - chensun
+  - connor-mccarthy
+  - gkcalat
+  - jlyaoyuli
+  - Linchin
 reviewers:
-  - Ark-kun
   - chensun
```

Comment:
Should we add Connor here too?

Reply:
I was thinking backend and frontend tests are more likely to be affected by external changes, so probably Connor's attention are less required.